### ISSUE TITLE
[FilterBar | a11y]: Warning message after click on Reset button is read by JAWS screen reader software for two times

### ISSUE THREAD

**Issue by marcelodebittencourt-sap** [SAP/ui5-webcomponents#9899]
### Bug Description

With JAWS Screen reader after click on "Adapt filters" button, de-select any option and then click on "Reset" button, the warning message and also the OK and Cancel button are read with voice for 2 times causing confusion to the user.

### Affected Component

CA-UI5-CMP

### Expected Behaviour

Warning message and buttons should be read only once by JAWS screen reader.

### Isolated Example

https://ui5.sap.com/1.128.1/#/entity/sap.ui.comp.filterbar.FilterBar/sample/sap.ui.comp.sample.filterbar.DynamicPageListReport

### Steps to Reproduce

1. Click on "Adapt Filters" button
2. De-select any item, for example "Category" 
3. Enable JAWS screen reader
4. Click on "Reset" button
5. Notice the Warning message "You are about to reset all the changes you made to your view settings and revert to the selected view." and OK and Cancel button are read 2 times

### Log Output, Stack Trace or Screenshots

https://github.com/user-attachments/assets/ad6cbd76-5ef2-435e-929a-16a24ce5ba44

### Priority

Medium

### UI5 Web Components Version

1.128.1

### Browser

Edge

### Operating System

Microsoft Windows 11 Enterprise Version 23H2

### Additional Context

Tested with JAWS Professional Edition Version 2024.2409.2

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @marcelodebittencourt-sap ,
The FilterBar is part of SAPUI5 not of UI5 WebComponents. This GitHub repository is a place for managing issues for UI5 WebComponents only.
Please, open a ticket in SAP ticket system for managing incidents.

Regards,
Iliana
